kuma/users/tests/test_tasks.py

```diff
@@ -63,6 +73,22 @@ def test_welcome_mail_for_verified_email(self):
         eq_(expected_to, welcome_email.to)
         ok_(u'utm_campaign=welcome' in welcome_email.body)
 
+    def test_signup_getting_started_message(self):
+        u = user(username='welcome',
```

Comment:
I tried to think of something more descriptive, but `user` is already the name of the helper function. This is repeated in many other tests, so I didn't feel obligated to fix it only here.